### PR TITLE
Donate to eff continued

### PR DESCRIPTION
**Pull request by Hainish**

### CODE REVIEW

File: chromium/popup.html [hunk 1]
```diff
@@ -60,7 +60,8 @@ <h3 i18n="chrome_experimental_rules"></h3>
 </div>
 
 <footer>
-    <a id="aboutTitle" href="https://www.eff.org/https-everywhere" target="_blank" i18n="about_title">blah</a>
+    <a id="aboutTitle" href="https://www.eff.org/https-everywhere" target="_blank" i18n="about_title">blah</a><br><br>
```
Comment (jsha) [inline chromium/popup.html]: Either leave this blank or fill in the default English value, rather than using "blah"

Comment (Hainish) [inline chromium/popup.html]: Ah, good point - I was just copying from the previously committed template.

**Comment by Hainish** [comment]
Ready for review again @jsha

**Comment by cooperq** [comment]
LGTM, ready to commit as soon as tests pass

**Comment by jsha** [comment]
LGTM

**Pull request merged**